### PR TITLE
First Pull

### PR DESCRIPTION
**Pull request by verasraul**
First pull request created for main project (verasraul/PyCode/Projects/PasswordManager).

### CODE REVIEW

**Comment by verasraul** [comment]
@agaro1121

**Comment by agaro1121** [comment]
Nice!

You can make the description nice with some mark down.
Try something like this:

# Overview
- First pull request created for main project (verasraul/PyCode/Projects/PasswordManager).

# Design/Changes
- Adds Readme

File: README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# PyCode
+# Test branch for all python projects.
+
+
+# Main Project Name: PasswordManager.
+# Main Project Location: verasraul/PyCode/Projects/PasswordManager.
```
Comment (agaro1121) [inline README.md:6]: Some comment

File: README.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# PyCode
+# Test branch for all python projects.
```
Comment (agaro1121) [inline README.md:2]: This comment is for this line of code

**Comment by verasraul** [comment]
Copy.

**Review by agaro1121 — APPROVED** [review]

**Pull request merged**